### PR TITLE
[Teva-2660] delete pentest environment

### PR DESCRIPTION
**Pull request by felixtheflex**
## Jira ticket URL

- Just add the ticket number to the end:

https://dfedigital.atlassian.net/browse/TEVA-2660

## Changes in this PR:

Delete Pentest environment
  .github/workflows/deploy_branch.yml
  .github/workflows/deploy_app.ym
  terraform/workspace-variables/pentest.tfvars

Add terraform-app-destroy to Makefile
  Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -129,6 +129,9 @@ terraform-app-destroy-review: terraform-app-init ## make passcode=MyPasscode CON
 		$(if $(CONFIRM_DESTROY), , $(error Can only run with CONFIRM_DESTROY))
 		cd terraform/app && terraform destroy -var-file ../workspace-variables/review.tfvars -auto-approve
 
+terraform-app-destroy: terraform-app-init ## make qa destroy passcode=MyPasscode
```
Comment (saliceti) [inline Makefile:132]: This is useful but dangerous. Shall we add a safeguard like above?
`$(if $(CONFIRM_DESTROY), , $(error Can only run with CONFIRM_DESTROY))`

Comment (saliceti) [reply Makefile:132]: Even better, make terraform-app-destroy work for review as well

Comment (felixtheflex) [reply Makefile:132]: ....Correct, which is why i didnt put `-auto-approve`

**Review by saliceti — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (saliceti) [reply Makefile:122]: Explained IRL

**Pull request merged**